graphql/app/graphql/utils.js

```diff
@@ -0,0 +1,2 @@
+exports.getSelectionSet = info =>
```

Comment:
what are you doing here?

Reply:
this is part of the bootstrap to extract the selection set in an easier way from the graphql context